packages/components/src/components/hds/flyout/description.ts

```diff
@@ -6,9 +6,16 @@
 import Component from '@glimmer/component';
 import { deprecate } from '@ember/debug';
 
-export default class HdsFlyoutDescriptionComponent extends Component {
-  constructor() {
-    super(...arguments);
+interface HdsFlyoutDescriptionSignature {
+  Blocks: {
+    default: [];
+  };
+  Element: HTMLElement;
```

Comment:
Looking at the template, we know this to be a `<div>`; even better we can reference the  `HdsTextBodySignature['Element']` directly